pdns/test-packetcache_hh.cc

```diff
@@ -352,14 +349,8 @@ BOOST_AUTO_TEST_CASE(test_PacketCacheRecCollision) {
     opt.source = Netmask("192.0.2.1/32");
     ednsOptions.clear();
     ednsOptions.push_back(std::make_pair(EDNSOptionCode::ECS, makeEDNSSubnetOptsString(opt)));
-    EDNSCookiesOpt cookiesOpt;
-    cookiesOpt.client = string("deadbeef");
-    cookiesOpt.server = string("deadbeef");
-    cookiesOpt.server[4] = -20;
-    cookiesOpt.server[5] = -114;
-    cookiesOpt.server[6] = 0;
-    cookiesOpt.server[7] = 0;
-    ednsOptions.push_back(std::make_pair(EDNSOptionCode::COOKIE, makeEDNSCookiesOptString(cookiesOpt)));
+    EDNSCookiesOpt cookiesOpt(string("deadbeefdead\x11\xee\x00\x00").c_str(), 16);
```

Comment:
I was surprised by the new values, wondering how the test could still pass. It turns out that we don't need to compute a hash collision for this since the content of cookies is skipped in the current packet cache code, so that doesn't really matter :-)